Core/libMOOS/CMakeLists.txt

```diff
@@ -199,6 +193,13 @@ install(
     FILES_MATCHING PATTERN "*.h" PATTERN "*.hxx" PATTERN "*.hpp"
 )
 
+# Create and install ClientDefines.h header for ASYNCHRONOUS_CLIENT_DEFINE variable
+configure_file(${CMAKE_MODULE_PATH}/ClientDefines.h.in
```

Comment:
These paths should be quoted too. I know the originals weren't, and I know that it was probably me who wrote the originals, but you've got to start somewhere, right :-)

We can make a separate PR to fix the the rest of them, but it seems right to fix these one here while you're touching them.